raiden/tests/integration/long_running/test_integration_events.py

```diff
@@ -228,10 +228,6 @@ def run_test_channel_deposit(raiden_chain, deposit, retry_timeout, token_address
         registry_address, token_address, app0.raiden.address, app1.raiden.address, deposit
     )
 
-    wait_both_channel_deposit(app0, app1, registry_address, token_address, deposit, retry_timeout)
```

Comment:
Could you explain why you removed this?

Reply:
Actually i just moved it to the end, since we were making the deposit for the first app and the waiting but the waiting method waits for both deposits so it was failing for timeout, so now we make both deposits and then we wait, so now it make sense, before this it was working on develop since the key was only the channel id and not the address, but it was wrong and now we can see that, so to fix it i just moved that to the end to wait for both apps to make the deposits